spec/javascripts/views/story_link_view_spec.js

```diff
@@ -0,0 +1,66 @@
+var StoryLinkView = require('views/story_link_view');
+
+describe('StoryLinkView', function() {
+
+  beforeEach(function() {
+    StoryLinkView.prototype.noteTemplate = sinon.stub().returns('');
+    StoryLinkView.prototype.hoverBox = sinon.stub().returns('');
+
+    var $el = $(`<a class="StoryLink"></a>`);
+    var Story = Backbone.Model.extend({
+      description: '',
+      name: 'story',
+      url: '/foo',
+      views: [{highlight: sinon.stub()}]
+    });
+
+    this.story = new Story({id: 2, title: 'StoryTitle', description: ''});
+    this.view = new StoryLinkView({model: this.story, el: $el});
+  });
+
+  it("has a data-original-title attribute", function() {
+    expect(this.view.$el.data('original-title')).toBe(this.story.get('title'));
```

Comment:
Prefer to pass the title hardcoded here, it's safer, we can't guarantee that creating the `StoryLinkView` instance won't change the model. You can do like this:

```javascript
expect(this.view.$el.data('original-title')).toBe('StoryTitle');
```